### ISSUE TITLE
v3.2-beta1-sdk1: jmonkeyplatform-linux-x64.sh seems to be corrupted

### ISSUE THREAD

**Issue by forenbuddy** [jMonkeyEngine/sdk#152]
I downloaded this file several times on different computers and through different providers. All those files are identical and all give this error message when started.

My system is Debian Stretch, 64 bit.

Full log:

```
/downloads/jmonkey ./jmonkeyplatform-linux-x64.sh 
Configuring the installer...
Searching for JVM on the system...
Preparing bundled JVM ...
Extracting installation data...

Installer file /downloads/jmonkey/jmonkeyplatform-linux-x64.sh seems to be corrupted

```

The 32 bit file works, but I want the 64 bit installation.

**Comment by grizeldi** [comment]
There's supposed to be a 3.2.0-stable release sometime soon, when @MeFisto94 manages to get travis to build it, I'd only worry if that one was corrupted too.

**Comment by MeFisto94** [comment]
This is now solved as there is the v3.2.0-stable-sdk1 release and the v3.2-beta1-sdk1 release has been removed permanently.

Btw: What happened was that the travis disk space was exhausted and so it uploaded a half-done file.
